src/js/utils/dom.js

```diff
@@ -0,0 +1 @@
+export const $ = (selector) => document.querySelector(selector);
```

Comment:
```suggestion
export const $ = (selector, { element }) => { 
  if (element != null) {
    return element.querySelector(selector)
  }
  
  return document.querySelector(selector); 
}
```

좀 더 나아가 본다면, 요소를 받아서 좀 더 좁은 범위에서 querySelector를 수행할 수도 있을 거에요

Reply:
document부터 찾으면 상황에 따라 비용이 클 수 있으니 요소를 지정해서 querySelector를 수행할 수 있도록 함수를 작성하는 것도 좋은 것 같습니다!